Remake header.(#7)

asgeneral opened the pull request:
Add button, readable css properties and method check authorise.
![header](https://user-images.githubusercontent.com/35890477/35732141-ee91aaca-084a-11e8-9583-91af1109c558.png)

asgeneral committed:
Header carcass is completed

asgeneral committed:
Header carcass is completed

asgeneral committed:
Header carcass is completed

asgeneral committed:
Header carcass is completed

asgeneral committed:
Created header without connection to server

asgeneral committed:
Created header without connection to server

asgeneral committed:
Header completed.(#7)

asgeneral committed:
Header complited.(#7)

lbdlbdlbdl commented on src/app/app.module.ts:
```diff
@@ -3,6 +3,9 @@ import { environment } from '../environments/environment';
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import {MatToolbarModule} from '@angular/material/toolbar';
```
Советую добавить material модули строками ниже после комментария "Material modules", как сделано ниже для firebase модулей. Слишком много неупорядоченных ссылок на модули делают навигацию по файлу неудобной.

bondiano committed:
Merge branch 'master'
# Conflicts resolved:
#	src/app/app.module.ts

bondiano committed:
Fix all for header (#7)
Move all file to it necessary place
Update angular md dependance
Fix for PR requests

bondiano commented:
![image](https://user-images.githubusercontent.com/323479/35763630-01cf7e1c-08e3-11e8-8a5c-c342552dba7a.png)
После изменений
